### PR TITLE
DOC-751: Added documentation for new bootstrap icon pack

### PR DESCRIPTION
**Pull request by lnewson**
Related Ticket: DOC-751

Description of Changes:
* Adds a release note for the new bootstrap icon pack.
* Updates the skin/icon compatibility matrix.
* Updates the bootstrap demo to use the bootstrap icon pack (won't work until it's on cloud though). Also fixed some mino syntax issues in the demo.

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

**Review by ashfordneil — APPROVED** [review]

File: enterprise/premium-skins-and-icon-packs/bootstrap-demo.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ description: Bootstrap Demo
 keywords: skin skins icon icons bootstrap customize theme
 ---
 
-This demo shows an example form using Bootstrap together with {{site.productname}} with the _Bootstrap_ skin. It's designed to closely matches _colors, forms and buttons_ to the default Bootstrap framework
+This demo shows an example form using Bootstrap together with {{site.productname}} with the _Bootstrap_ skin and _Bootstrap_ icon pack. It's designed to closely matches _colors, forms and buttons_ to the default Bootstrap framework
```
Comment (tylerkelly13) [inline enterprise/premium-skins-and-icon-packs/bootstrap-demo.md:9]:
```suggestion
This demo shows an example form using Bootstrap together with {{site.productname}} and the {{site.companyname}} _Bootstrap_ skin and _Bootstrap_ icon pack. It's designed to closely match the _colors, forms, and buttons_ of the default Bootstrap framework UI.
```
Maybe?

File: release-notes/release-notes57.md [hunk 2]
```diff
@@ -83,6 +84,12 @@ For information on:
 - The `spellchecker_ignore_list` option, see: [Spell Checker Pro - `spellchecker_ignore_list`]({{ site.baseurl }}/plugins/premium/tinymcespellchecker/#spellchecker_ignore_list).
 - The `addIgnoredWords` API, see: [Spell Checker Pro - `API`]({{ site.baseurl }}/plugins/premium/tinymcespellchecker/#api).
 
+## Accompanying Premium Skins and Icon Packs changes
+
+The {{site.productname}} 5.7 release includes an accompanying release of the **Premium Skins and Icon Packs** premium content. Included is a new `bootstrap` icon pack that provides [Bootstrap icons](https://icons.getbootstrap.com/) for {{site.productname}}.
```
Comment (tylerkelly13) [inline release-notes/release-notes57.md:89]:
```suggestion
The {{site.productname}} 5.7 release includes an accompanying release of the **Premium Skins and Icon Packs**. Included is a new `bootstrap` icon pack that provides [Bootstrap icons](https://icons.getbootstrap.com/) for {{site.productname}}.
```

**Review by tylerkelly13 — APPROVED** [review]
A couple minor edits needed, otherwise, LGTM

File: release-notes/release-notes57.md (hunk 2 above)
Comment (lnewson) [reply release-notes/release-notes57.md:89]: Hehe, I did originally have that but then added the additional part to keep it consistent with the previous uses: https://www.tiny.cloud/docs/release-notes/release-notes52/#premiumskinsandiconpacks12 (yes this says `premium plugin` which is wrong).

File: enterprise/premium-skins-and-icon-packs/bootstrap-demo.md (hunk 1 above)
Comment (lnewson) [reply enterprise/premium-skins-and-icon-packs/bootstrap-demo.md:9]: If you're changing that you'll need to change all premium skin demos, as they are all the same as my original wording.

Comment (tylerkelly13) [reply enterprise/premium-skins-and-icon-packs/bootstrap-demo.md:9]: :facepalm: :vomiting_face: Up to you. ether accept my change so at least one is right, or ignore me and I'll fix them all one day.

File: release-notes/release-notes57.md (hunk 2 above)
Comment (tylerkelly13) [reply release-notes/release-notes57.md:89]: :slightly_frowning_face:  that's probably my bad. ignore previous ones, this is better.

**Review by tylerkelly13 — APPROVED** [review]

**Review by EkimChau — APPROVED** [review]

**Pull request merged**